### ISSUE TITLE
File get corrupted if ctrl+s then closed the program

### ISSUE THREAD

**Issue by mohalidev** [jgraph/drawio-desktop#228]
**** When I close the program, the file I'm working on bcomes 0kb on windows.

I lost an important diagram when I ctrl+s a program. I am now wary and I keep copying the entire folder with every save. This is completely unreliable. Thanks.

**Comment by davidjgraph** [comment]
What version of the desktop build were you running when that happened?

**Comment by JPraeses** [comment]
I have encountered this issue (or a similar issue?) using version:
draw.io-12.6.5-windows-no-installer.exe

I was working with a .drawio file that was at least 460 KB.
The program started freezing up.  So I tried to save my changes.  It would briefly "unfreeze" but then "freeze" again.  I tried to save my changes.  Eventually I closed the program.  I don't think I had to use the Windows "End Task" screen.

I restarted the program and when Itried to open the .drawio file I got the message "Error loading file" "Not a diagram file".  After that is when I saw that the file had 0 KB.
